api/cart.mjs

```diff
@@ -0,0 +1,79 @@
+import Postmates from "../services/Postmates.mjs";
+import { HTTPResponseError } from "../errors/http.mjs";
+
+/* Search postmates
+ * @param {item} postmates item{} used to initailize cart
+ * @return {object} either success json or error
+ */
+const createPostmatesCart = async (item, itemQuantity) => {
+  try {
+    const postmates = new Postmates();
+    return (await postmates.createCart(item, itemQuantity)).data;
+  } catch (e) {
+    if (e instanceof HTTPResponseError) {
+      return { error: await e.getError() };
+    } else {
+      console.error(e);
+    }
+  }
+};
+
+const addToPostmatesCart = async (item) => {
```

Comment:
Let's add some function docs (params, return, descriptions). Same for the other functions in this file.